server/routes/authRouter.js

```diff
@@ -139,7 +139,7 @@ authRouter.route('/').get((req, res) => {
 });
 
 authRouter.get('/logout', function(req, res) {
-  console.log('logout auth:', req.isAuthenticated(), req.session);
+  // console.log('logout auth:', req.isAuthenticated(), req.session);
```

Comment:
If we can start writing tests, and those are fine, I think we can take off these console log message real soon.